packages/wrangler/src/config/validation.ts

```diff
@@ -3159,6 +3159,43 @@ const validateConsumer: ValidatorFn = (diagnostics, field, value, _config) => {
 	return isValid;
 };
 
+const isValidDate = (value: string): boolean => {
+	const data = new Date(value);
+	return !isNaN(data.getTime());
+};
+
+const validateCompatibilityDate: ValidatorFn = (diagnostics, field, value) => {
```

Comment:
let's make sure we surface all errors on first throw, as opposed to one per cmd run. See my comment [here](https://github.com/cloudflare/workers-sdk/blob/main/packages/wrangler/src/config/validation.ts#L2083-L2084). It makes for a much better UX and it's something we're pushing for since recently.

[validateAssetsConfig](https://github.com/cloudflare/workers-sdk/blob/main/packages/wrangler/src/config/validation.ts#L2067) is a good example of how'd like things to work

Reply:
✅